api/v1beta1/cluster_types.go

```diff
@@ -624,12 +625,14 @@ type WorkersTopology struct {
 	// +optional
 	// +listType=map
 	// +listMapKey=name
+	// +kubebuilder:validation:MaxItems=1000
```

Comment:
I just remembered some user, probably in the telco field, bringing up they have many single node MD, let's bump this to 
```suggestion
	// +kubebuilder:validation:MaxItems=2000
```

Reply:
Good point. Done